packages/password/src/accounts-password.ts

```diff
@@ -256,18 +251,14 @@ export default class AccountsPassword implements AuthenticationService {
    * @returns {Promise<void>} - Return a Promise.
    */
   public async sendVerificationEmail(address: string): Promise<void> {
-    if (!address) {
+    if (!address || !isString(address)) {
       throw new Error('Invalid email');
     }
+
     const user = await this.db.findUserByEmail(address);
     if (!user) {
       throw new Error('User not found');
     }
-    // Make sure the address is valid
```

Comment:
Here we actually don't need this check since we are calling `this.db.findUserByEmail` to get the user.